diff --git a/node_modules/fastify-sse-v2/lib/plugin.js b/node_modules/fastify-sse-v2/lib/plugin.js
index 1054757..d09a8d5 100644
--- a/node_modules/fastify-sse-v2/lib/plugin.js
+++ b/node_modules/fastify-sse-v2/lib/plugin.js
@@ -31,7 +31,7 @@ const plugin = function (instance, options) {
                 this.raw.setHeader("Connection", "keep-alive");
                 this.raw.setHeader("Cache-Control", "no-cache,no-transform");
                 this.raw.setHeader("x-no-compression", 1);
-                this.raw.write((0, sse_1.serializeSSEEvent)({ retry: options.retryDelay || 3000 }));
+                this.raw.write((0, sse_1.serializeSSEEvent)({}));
                 handleAsyncIterable(this, this.sseContext.source);
             }
             if ((0, util_1.isAsyncIterable)(source)) {
diff --git a/node_modules/fastify-sse-v2/lib/sse.js b/node_modules/fastify-sse-v2/lib/sse.js
index a8240ca..ee6aa79 100644
--- a/node_modules/fastify-sse-v2/lib/sse.js
+++ b/node_modules/fastify-sse-v2/lib/sse.js
@@ -36,7 +36,6 @@ function transformAsyncIterable(source) {
             }
             finally { if (e_1) throw e_1.error; }
         }
-        yield yield __await(serializeSSEEvent({ event: "end", data: "Stream closed" }));
     });
 }
 exports.transformAsyncIterable = transformAsyncIterable;
